docs/03-Filtering Patches.md

```diff
@@ -1,18 +1,51 @@
 ## Filtering Patches
 
-You can ignore local changes by setting `Palindrom.filterLocalChange` function. This function should accept a JSON Patch operation and either return it or return a falsy value. Obviously, returning a falsy value will result an ignored operation.
+### 1. By defining non-enumerable properties _(recommended)_:
+Assuming your Palindrom instance is instantiated and your `palindrom.obj` is ready, you can create a non-enumerable object child in `palindrom.obj` and use it as a local non-synced store for your needs.
+
+#### Example:
+
+```js
+Object.defineProperty(palindrom.obj, "myLocalNonSyncedObject", {
+    enumerable: false,
+    value: {},
+    writable: true
+});
+```
+Now `palindrom.obj.myLocalNonSyncedObject` is an object this is not synced with the server. This is ideal for keeping client-side related information (eg: div height, or something like `isModalWarningDisplayed = true`).
```

Comment:
I would change "not synced with the server" to "not observed for patch generation".